java-runtime-builder-app/src/main/java/com/google/cloud/runtimes/builder/buildsteps/PrebuiltRuntimeImageBuildStep.java

```diff
@@ -54,7 +63,31 @@ protected String getArtifact(BuildContext buildContext) throws BuildStepExceptio
     } else if (artifacts.size() > 1) {
       throw new TooManyArtifactsException(artifacts);
     } else {
-      return buildContext.getWorkspaceDir().relativize(artifacts.get(0)).toString();
+      Path artifact = artifacts.get(0);
+      logger.debug("Found Java artifact {}", artifact);
```

Comment:
I'm not sure why, but when I run this against an exploded war with `WEB-INF/appengine-web.xml` it outputs this line twice:
```
DEBUG - Found Java artifact /workspace
DEBUG - Found Java artifact /workspace
```